pkg/model/misc.go

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Convenience functions are good.. and I see when they are useful.. 

But I have to gripes here..

### misc.go

Just naming a file that, opens up the door to contributors dropping in arbitrary logic they don't want to figure out where it should really go

Suggest: `convenience.go` 

### s() and i64()

While these are super handy, and I love them in the new models.. they don't really tell the dev (usually me) what the heck they do..

(Yes I mean this) Can we maybe use the convenience functions as wrappers to very specific verbose function names?

```
func s(v string) *string {
    return stringToStringPointer(v)
}

func stringToStringPointer(v string) *string {
    return &v
}
```

It might seem like super over kill to dig 3 functions deep just to add a `&` to the beginning of a string variable name, but pointers are a **huge** deal. Also when we are working on the models, if we need to see what `s()` does, the code will document itself immediately.